rest/server_context.go

```diff
@@ -411,12 +411,12 @@ func (sc *ServerContext) _getOrAddDatabaseFromConfig(config *DbConfig, useExisti
 
 	bucket, err := db.ConnectToBucket(spec, func(bucket string, err error) {
 
-		msg := fmt.Sprintf("%v dropped Mutation feed (TAP/DCP) due to error: %v, taking offline", bucket, err)
-		base.Warn(msg)
+		msgFormatStr := "%v dropped Mutation feed (TAP/DCP) due to error: %v, taking offline"
+		base.WarnR(fmt.Sprintf(msgFormatStr, base.UD(bucket), err))
 
 		if dc := sc.databases_[dbName]; dc != nil {
 
-			err := dc.TakeDbOffline(msg)
+			err := dc.TakeDbOffline(fmt.Sprintf(msgFormatStr, "Bucket", err))
```

Comment:
I guess `"Bucket"` here could be the actual bucket variable

Reply:
Fixed